make test fails on 1.4.0 version

```
$ make test
env NAME=osixia/openldap VERSION=1.4.0 bats test/test.bats
 ✓ image build
 ✓ ldapsearch new database
 ✓ ldap domain with non-matching ldap base dn
 ✓ ldap domain with matching ldap base dn subdomain
 ✓ ldap base dn domain with matching ldap subdomain
 ✓ ldap domain with ldap base dn subdomain included
 ✗ ldapsearch database from created volumes
   (from function `wait_process_by_cid' in file test/test_helper.bash, line 80,
    in test file test/test.bats, line 92)
     `wait_process_by_cid $LDAP_CID slapd' failed with status 143
   9b14b2ea373096b8cc6418e4580dc2f1bb87de4eacd21c01167b0a6df1b92f10
 ✗ ldapsearch database with password provided from file
   (in test file test/test.bats, line 120)
     `[ "$status" -eq 0 ]' failed
 ✗ ldapsearch new database with strict TLS
   (in test file test/test.bats, line 134)
     `[ "$status" -eq 0 ]' failed
 ✓ ldapsearch new database with strict TLS and custom ca/crt
 ✓ ldapsearch new database with strict TLS and custom ca/crt and custom dhparam
 ✓ ldapsearch existing hdb database and config
 ✗ replication with new databases and strict TLS
   (in test file test/test.bats, line 205)
     `docker exec $LDAP_REPL_CID ldapadd -x -D "cn=admin,dc=example,dc=org" -w admin -f /container/service/slapd/assets/test/new-user.ldif -h ldap2.example.org -ZZ' failed
   ldap_start_tls: Connect error (-11)
   	additional info: (unknown error code)

13 tests, 4 failures

Makefile:13: recipe for target 'test' failed
```

--- a/image/Dockerfile
+++ b/image/Dockerfile
@@ -1,6 +1,6 @@
 # Use osixia/light-baseimage
 # sources: https://github.com/osixia/docker-light-baseimage
-FROM osixia/light-baseimage:1.2.0
+FROM osixia/light-baseimage:1.3.1
 
 ARG LDAP_OPENLDAP_GID
 ARG LDAP_OPENLDAP_UID

